roles/post-db-restore/tasks/main.yml

```diff
@@ -136,7 +136,7 @@
 
 # Copy public files from AFS so that they exist and Drupal can find them if needed
 - name: Copy public files from AFS to Site Factory
-  shell: "drush -y rsync '/afs/ir/dist/drupal/{{ site_prefix }}_{{ inventory_hostname }}/files/' @acsf{{ drush_environment }}.{{ stack }}.{{ acsf_site_name }}:%files/ --exclude-paths=sites/default/files/js_injector/"
+  shell: "drush -y rsync '/afs/ir/dist/drupal/{{ site_prefix }}_{{ inventory_hostname }}/files/' @acsf{{ drush_environment }}.{{ stack }}.{{ acsf_site_name }}:%files/ --exclude-paths=sites/default/files/js_injector/ --exclude-paths=sites/default/files/css/ --exclude-paths=sites/default/files/js/"
```

Comment:
Shouldn't the `--exclude-paths` come before the rsync option?
https://drupal.stackexchange.com/questions/178224/clear-example-for-drush-rsync-exclude-paths

Reply:
Like I'm going to believe what Greg Anderson has to say about `drush rsync`

Reply:
Also, shouldn't the options be separated by `:`

Reply:
How do you mean?

Reply:
```suggestion
  shell: "drush -y  --exclude-paths=sites/default/files/js_injector/ --exclude-paths=sites/default/files/css/:sites/default/files/js/ rsync '/afs/ir/dist/drupal/{{ site_prefix }}_{{ inventory_hostname }}/files/' @acsf{{ drush_environment }}.{{ stack }}.{{ acsf_site_name }}:%files/"
```

Reply:
I wasn't aware of that shorthand. Why not all three?

Reply:
Gotta leave something for you.

Reply:
Can't argue with that logic.